.github/workflows/replay-verify.yaml

```diff
@@ -23,11 +23,7 @@ on:
       - ".github/workflows/replay-verify.yaml"
       - "testsuite/replay_verify.py"
   schedule:
-    - cron: "0 22 * * *"
-  push:
-    branches:
-      - aptos-release-v* # the aptos release branches
-      - aptos-node-v*
```

Comment:
to stop this from running on push on these specific branches, we'll need to cherry-pick this into those branches too.